lte/gateway/python/load_tests/loadtest_pipelined.py

```diff
@@ -0,0 +1,199 @@
+#!/usr/bin/env python3
+
+"""
+Copyright 2020 The Magma Authors.
+
+This source code is licensed under the BSD-style license found in the
+LICENSE file in the root directory of this source tree.
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+import os
+import json
+import argparse
+import random
+import subprocess
+from collections import namedtuple
+
+from google.protobuf import json_format
+from lte.protos.pipelined_pb2 import (
+    ActivateFlowsRequest,
+    DeactivateFlowsRequest,
+    RequestOriginType,
+    VersionedPolicy,
+    VersionedPolicyID,
+)
+from lte.protos.policydb_pb2 import (
+    FlowMatch,
+    FlowDescription,
+    PolicyRule,
+)
+from lte.protos.subscriberdb_pb2 import AggregatedMaximumBitrate
+from magma.pipelined.policy_converters import convert_ipv4_str_to_ip_proto
+from magma.pipelined.qos.common import QosManager
+from magma.subscriberdb.sid import SIDUtils
+
+PROTO_DIR = 'lte/protos'
+IMPORT_PATH = '/home/vagrant/magma'
+RESULTS_PATH = '/var/tmp'
+
+UEInfo = namedtuple('UEInfo', ['imsi_str', 'ipv4_src', 'ipv4_dst',
+                               'rule_id'])
+
+def _gen_ue_set(num_of_ues):
+    imsi = 123000000
```

Comment:
can we have single definition between load-test and pipelined_cli ?

Reply:
I can probably just import it, will fix

Reply:
sure let me import it